apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -838,6 +886,99 @@ int _wt_parse_power(struct wt_options *opt, int argc, char *argv[])
 	return -1;
 }
 
+int _wt_parse_add_hostname(struct wt_options *opt, int argc, char *argv[])
+{
+ 	if (argc != 5) {
+		return -1;
+	}
+	opt->path = argv[3];
```

Comment:
Is it okay not to check that `opt` is NULL or not?

Reply:
Sir, other parse functions of wm_test.c files are implemented without NULL checking. So, I followed similar style.